fontforge/python.c

```diff
@@ -2852,6 +2852,10 @@ static int PyFFContour_SubAssign( PyObject *self, PyObject *key, PyObject *val )
     if ( PyLong_Check(key)) {
         return PyFFContour_IndexAssign(self, PyNumber_AsSsize_t(key, PyExc_IndexError), val);
     }
+    if ( val==NULL ) {
+        PyErr_Format(PyExc_TypeError, "Deleting multiple points is not supported.");
```

Comment:
Deletion with any non-numeric key (even just a mistaken argument) would trigger this message. Maybe differentiate:
- valid range --> delete range
- invalid range --> invalid range error
- not range --> type error for key argument